docs/plugins/spectate.rst

```diff
@@ -11,7 +11,30 @@ Usage
 
 ::
 
-    enable spectate
```

Comment:
if `spectate` shouldn't be directly enabled, then the plugin shouldn't implement the `plugin_enabled` API. You can still have the function, just don't name it `plugin_enabled` and don't export it.

Reply:
It can be, should I just remove the spec enable ordering? I guess the enable x would obviously be the preferred choice. I'll remove the enable/disable parameters

Reply:
I suggested a different API in another comment that would sidestep this issue, I think.